databases/business-sync/fragment-shoebox-snippet.js

```diff
@@ -0,0 +1,40 @@
+{
+  channels: getDocSyncChannels(doc, oldDoc),
+  typeFilter: function(doc, oldDoc) {
+    return createBusinessEntityRegex('shoeboxSnippet\\.[A-Za-z0-9_-]+\\.[A-Za-z0-9_-]+').test(doc._id);
```

Comment:
Personally, I like the metaphor as it fits with our 'story' - maybe not a bad thing to have to explain to every new dev.  Personal preference, really, but I could get behind `shoeboxItem`.